samples/sample_capture_async.py

```diff
@@ -0,0 +1,61 @@
+"""Sample demonstrating how to capture with multiple cameras at the same time."""
+import concurrent.futures
+import datetime
+import time
+
+import zivid
+
+
+def _sync_version(cameras: list[zivid.Camera]) -> list[zivid.Frame]:
```

Comment:
Maybe nicer to name these `_capture_async` and `_capture_sync` or maybe `_(a)sync_capture`

Reply:
Amended